tests/scripts/ci.py

```diff
@@ -24,12 +24,15 @@
 import getpass
 import inspect
 import argparse
+import json
+import shutil
 import grp
 import subprocess
 from pathlib import Path
 from typing import List, Dict, Any, Optional
 
 REPO_ROOT = Path(__file__).resolve().parent.parent.parent
+SCRIPT_DIR = REPO_ROOT / ".cipy-scripts"
```

Comment:
should we call it `ci-py-scripts` or something, so that it doesn't look like a `cpy` typo?